Appprogrammering/app/src/main/java/com/example/appprogrammering2/secondActivity.kt

```diff
@@ -0,0 +1,56 @@
+
+package com.example.appprogrammering2
+
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.content.Context
+import android.content.Intent
+import android.os.Build
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.Button
+import androidx.core.app.NotificationCompat
+import androidx.core.app.NotificationManagerCompat
+import kotlinx.android.synthetic.main.activity_second.*
+
+class secondActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_second)
+        val channelID = "testapp" //notification channel id
+
+        var counter = 0
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
```

Comment:
Hvorfor er dette nødvendigt?

Reply:
For at sørge for at sdk versionen er den rigtige